source/material_model/rheology/visco_plastic.cc

```diff
@@ -246,7 +268,10 @@ namespace aspect
 
             if (this->get_parameters().enable_elasticity)
               {
-                const std::vector<double> &elastic_shear_moduli = elastic_rheology.get_elastic_shear_moduli();
+                // Step 3a: calculate viscoelastic (effective) viscosity
+                // The preyield viscoelastic viscosity is scaled with the
+                // timestep ratio in the calculate_viscoelsatic_viscosity function.
```

Comment:
Not sure this is the right place for this comment. It is an implementation details that would belong in the `calculate_viscoelastic_viscosity` function, in particular because the meaning of timestep ratio is not explained.